src/AuthService.Grpc/Infrastructure/Configurations/AppConfigurations.FluentValidation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+
+namespace AuthService.Grpc.Infrastructure.Configurations
+{
+    public static partial class AppConfigurations
+    {
+        /// <summary>
+        /// Add validators.
+        /// </summary>
+        /// <param name="services"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddValidators(this IServiceCollection services)
```

Comment:
Please set extension method name according to Solution Architecture naming for fluent validation.